src/ontology/cl-edit.owl

```diff
@@ -32739,6 +32741,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:38020069") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4042036 "melanin-concentrating hormone neuron"@en)
 SubClassOf(obo:CL_4042036 obo:CL_2000029)
 
+# Class: obo:CL_4042039 (caudal ganglionic eminence derived neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12593982") Annotation(oboInOwl:hasDbXref "PMID:12637172") Annotation(oboInOwl:hasDbXref "PMID:20130169") Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK98190/") obo:IAO_0000115 obo:CL_4042039 "A neuron of the central nervous system that develops from a caudal ganglionic eminence.")
+AnnotationAssertion(terms:contributor obo:CL_4042039 <https://orcid.org/0000-0002-0098-8958>)
+AnnotationAssertion(terms:date obo:CL_4042039 "2024-11-14T11:27:25Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:12593982") Annotation(oboInOwl:hasDbXref "PMID:12637172") Annotation(oboInOwl:hasDbXref "PMID:20130169") Annotation(oboInOwl:hasDbXref "https://www.ncbi.nlm.nih.gov/books/NBK98190/") rdfs:comment obo:CL_4042039 "Neurons derived from the caudal ganglionic eminence consist mostly of neurons populating the cortical layers of the brain and some subcortical areas such as the globus pallidus, the striatus and the hippocampus. Amongst neurons derived from the caudal ganglionic eminence there are cortical GABAergic neurons such as Pax6, Lamp5, VIP, sncg, GABAergic; in the hippocampus, mossy fibers, pyramidal neurons, granule neurons and interneurons; in the striatum and globus pallidus, medium spiny projection neurons.")
+AnnotationAssertion(rdfs:label obo:CL_4042039 "caudal ganglionic eminence derived neuron"@en)
+SubClassOf(obo:CL_4042039 obo:CL_2000029)
+SubClassOf(obo:CL_4042039 ObjectSomeValuesFrom(obo:RO_0002202 obo:UBERON_0004026))
+
```

Comment:
@AvolaAmg This should be an EC def - in order to drive automated classification.